dev/source/docs/mavlink-requesting-data.rst

```diff
@@ -252,3 +262,16 @@ If using MAVProxy:
 
  - module load messagerate
  - messagerate status
+
+Rates Reverting to some Fixed value
```

Comment:
should we re-reference the LUA script since it constantly resets the rates if they change?

Reply:
I've haven't done exactly this but I've added an extra sentence to the Lua script section explaining why they might see variable rates when using the Lua script due to a tug-of-war with a GCS.